src/plugins/irc-events/link.js

```diff
@@ -98,26 +93,36 @@ function parseHtml(preview, res, client) {
 					preview.body = preview.body.substr(0, 300);
 				}
 
-				// Make sure thumbnail is a valid and absolute url
-				if (thumb.length) {
-					thumb = normalizeURL(thumb, preview.link) || "";
-				}
+				if (Helper.config.prefetchStorage || !Helper.config.disableMediaPreview) {
```

Comment:
This could be an early return instead of putting all the thumb code inside the `if` for less nesting.